masterpage/athleteRecords.php

```diff
@@ -1922,293 +1857,230 @@
-                }
-
-
-                //Gets the addresss
-                //State and City must be provided if zip is not provided. Address Line 1 must be provided
-                if (!isEmpty(colEntries[4]) && (!isEmpty(colEntries[8]) || (!isEmpty(colEntries[6]) && !isEmpty(colEntries[7])))) {
-                    var address = colEntries[4] + (isEmpty(colEntries[5]) ? "" : ", <br>") + colEntries[5] + (isEmpty(colEntries[6]) ? "" : ", <br>") + colEntries[6] + (isEmpty(colEntries[7]) ? "" : ", ") + colEntries[7] + (isEmpty(colEntries[8]) ? "" : ", ") + colEntries[8] + (isEmpty(colEntries[9]) ? "" : ", ") + colEntries[9];
-                    document.getElementById("address").innerHTML = address;
-                    var address = colEntries[4] + (isEmpty(colEntries[6]) ? "" : ", ") + colEntries[6] + (isEmpty(colEntries[7]) ? "" : ", ") + colEntries[7] + (isEmpty(colEntries[8]) ? "" : ", ") + colEntries[8] + (isEmpty(colEntries[9]) ? "" : ", ") + colEntries[9];
-                    document.getElementById("hrefAddress").href = "http://maps.google.com/?q=" + address;
-                }
-
-                //Populated dob, age, gender, tsize, dso, regNum, bibNum, memNum, ipcNum, teamName, hcName, hcPhone, hcEmail. In that order
-                for (var j = 12; j <= 26; j++) {
-                    if (!isEmpty(colEntries[j]))
-                        document.getElementById(colNames[j]).innerHTML = colEntries[j];
-                }
-
-                //Gets the phone and email of the athlete and places them as a link
-                if (!isEmpty(colEntries[23])) {
-                    document.getElementById("hcPhone").innerHTML = phoneFormat(colEntries[23]);
-                    document.getElementById("hrefHcPhone").href = "tel:" + colEntries[23];
-                }
-                if (!isEmpty(colEntries[24])) {
-                    document.getElementById("hrefHcEmail").href = "mailto:" + colEntries[24];
-                }
-
-                //Displays the sport section if selected
-                for (var j = 27; j < 36; j++) {
-                    if (!isEmpty(colEntries[j]) && colEntries[j] == 1) {
-                        document.getElementById("noSports").style.display = "none";
-                        document.getElementById(colNames[j]).style.display = "block";
-                    }
+        
+        let lines = data.split(/\r\n|\n/); 
+        let colNames = []
+        let check = []
+        for(let i = 0; i < lines.length; i++){
+            colNames[i] = lines[i].split(',');
+        }
+        console.log(colNames)
+
+        for(let row = 0; row < 1; row++){
+            check[row] = lines[row].split(',');
+            for(let column = 0; column < check[0].length; column++){
+                if(check[row][column].toLowerCase() == "regfor"){
+                    regIndex = column;
+                }else if(check[row][column].toLowerCase() == "firstname"){

```

Comment:
check[row][column].toLowerCase()

in CSV files, if any cell has a space in it, it will be surrounded by quotes. so you will have to take care of that. for example: first name will be stored as "first name", and check[row][column] will return the item with quotes and your comparison will fail